File: src/main/java/com/ject/studytrip/dummy/application/dto/DummyStampsInfo.java
package com.ject.studytrip.dummy.application.dto;

import java.util.List;

public record DummyStampsInfo(List<DummyStampInfo> stampsInfos) {
    public static DummyStampsInfo of(List<DummyStampInfo> stampsInfos) {
        return new DummyStampsInfo(stampsInfos);
    }
}

File: src/main/kotlin/com/ject/studytrip/dummy/application/dto/DummyStampsInfo.kt
package com.ject.studytrip.dummy.application.dto

data class DummyStampsInfo(
    val stampsInfos: List<DummyStampInfo>,
) {
    companion object {
        @JvmStatic
        fun of(stampsInfos: List<DummyStampInfo>): DummyStampsInfo = DummyStampsInfo(stampsInfos)
    }
}
